src/data/matrix/basic.lean

```diff
@@ -379,37 +381,39 @@ section distrib_mul_action
 variables [monoid R] [has_mul α] [add_comm_monoid α] [distrib_mul_action R α]
 
 @[simp] lemma smul_dot_product [is_scalar_tower R α α] (x : R) (v w : m → α) :
-  dot_product (x • v) w = x • dot_product v w :=
+  (x • v) ⬝ᵥ w = x • (v ⬝ᵥ w) :=
 by simp [dot_product, finset.smul_sum, smul_mul_assoc]
 
 @[simp] lemma dot_product_smul [smul_comm_class R α α] (x : R) (v w : m → α)  :
-  dot_product v (x • w) = x • dot_product v w :=
+  v ⬝ᵥ (x • w) = x • (v ⬝ᵥ w) :=
 by simp [dot_product, finset.smul_sum, mul_smul_comm]
 
 end distrib_mul_action
 
 section star_ring
 variables [semiring α] [star_ring α] (v w : m → α)
 
-lemma star_dot_product_star : dot_product (star v) (star w) = star (dot_product w v) :=
+lemma star_dot_product_star : (star v) ⬝ᵥ (star w) = star (w ⬝ᵥ v) :=
```

Comment:
You can remove some parens now that it's notation
```suggestion
lemma star_dot_product_star : star v ⬝ᵥ star w = star (w ⬝ᵥ v) :=
```
same in a few other places

Reply:
I removed some more parentheses. I noticed more places where they could be omitted but it looked weird without them, so I kept them in those places.